src/js/angular/clustermanagement/services/cluster-drawing.service.js

```diff
@@ -7,6 +7,13 @@ export const clusterColors = {
     ontoGrey: 'var(--gray-color)'
 };
 
+export const font = {
```

Comment:
Should be uppercase as a constant

Reply:
This is a cherry pick. This and the other comments will be fixed with the next cluster task.